api/spec/requests/api/address_books_spec.rb

```diff
@@ -0,0 +1,246 @@
+require 'spec_helper'
+
+module Spree
+  describe Api::AddressBooksController, :type => :request do
+    context 'as address book owner' do
+      context 'with ability' do
+        it 'returns my address book' do
+          state = create(:state)
+          harry_address_attributes = {
+            'firstname' => 'Harry',
+            'lastname' => 'Potter',
+            'address1' => '4 Privet Drive',
+            'address2' => 'cupboard under the stairs',
+            'city' => 'Surrey',
+            'zipcode' => '10010',
+            'phone' => '555-5555',
+            'state_id' => state.id,
+            'country_id' => state.country.id
+          }
```

Comment:
I'd prefer to see these attributes extracted to some let statements. In the case where I want to add a new test, I'd prefer to be able to access the existing values rather than copy/paste these around.